src/types/service_development_config.go

```diff
@@ -2,6 +2,7 @@ package types
 
 // ServiceDevelopmentConfig represents development specific configuration for a service
 type ServiceDevelopmentConfig struct {
-	Scripts map[string]string `yaml:",omitempty"`
-	Port    string            `yaml:",omitempty"`
+	HealthCheck string            `yaml:"health-check,omitempty"`
```

Comment:
Is it worth it to have this here just for consistency? We aren't using it for anything at the moment. @alexdavid thoughts?

Reply:
Probably YAGNI, at least in the near term. We can always add it if needed

Reply:
Alright @hugobho lets remove this.